docker/all-in-one/Dockerfile

```diff
@@ -230,4 +230,6 @@ HEALTHCHECK --interval=3s --timeout=2s --start-period=4s --retries=10 CMD [ "hea
 
 COPY docker/all-in-one/init /init
 COPY docker/all-in-one/entrypoint.sh /usr/local/bin/
+COPY docker/all-in-one/postgres-entrypoint.sh /usr/local/bin/
+COPY docker/all-in-one/shutdown.sh /usr/local/bin/supa-shutdown
```

Comment:
any reason not to preserve the `.sh` ext?

Reply:
no reason - will add extension